main.go

```diff
@@ -109,6 +109,10 @@ func main() {
 			Name:  "systemd-cgroup",
 			Usage: "enable systemd cgroup support, expects cgroupsPath to be of form \"slice:prefix:name\" for e.g. \"system.slice:runc:434234\"",
 		},
+		cli.IntFlag{
+			Name:  "systemd-stop-timeout",
```

Comment:
Should this be more flexible something like:
--systemd-options stop-timeout=50
Then it would be easier to add future values without having to add future options.

Reply:
I agree having a more extensible flag would be nicer.

Reply:
Sounds good :+1: I will update the PR to support that.